File: 17주차/SG/Height_Checker.cpp
#include<bits/stdc++.h>
using namespace std;
class Solution {
public:
    int heightChecker(vector<int>& heights) {
        vector<int> tmp = heights;
        sort(tmp.begin(),tmp.end());
        int answer=0;
        for(int i=0;i<heights.size();i++){
            if(tmp[i] != heights[i]){
                answer ++;
            }
        }    
        return answer;
    }
};

File: 17주차/NH/Height Checker.js
 function heightChecker(heights) {
    let sample=heights.slice(); //배열 복사 
    let count=0;
    sample.sort((a,b)=>a-b);
    for(let i=0; i<heights.length; i++)
        if(heights[i]!=sample[i])count++;
    return count;
};